### PR TITLE
Update Jenkins dependencies

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
https://github.com/radtriste/kogito-pipelines/pull/56 is an example of a PR

File: dsl/config/main.yaml [hunk 1]
```diff
@@ -14,41 +14,18 @@ ecosystem:
...
     default: main
 seed:
   config_file:
     git:
       repository: kogito-pipelines
       author:
-        name: kiegroup
-        credentials_id: kie-ci
-      branch: main
+        name: radtriste
```
Comment (mbiarnes) [inline dsl/config/main.yaml]: @radtriste changes for testing

**Review by mbiarnes — APPROVED** [review]

File: dsl/config/main.yaml (hunk 1 above)
Comment (radtriste) [reply dsl/config/main.yaml]: reverted

**Review by winklerm — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**